unitylibs/core/workflow/workflow-acrobat/action-binder.js

```diff
@@ -535,6 +547,9 @@ static ERROR_MAP = {
     this.transitionScreen = new TransitionScreen(this.transitionScreen.splashScreenEl, this.initActionListeners, this.LOADER_LIMIT, this.workflowCfg);
     await this.transitionScreen.showSplashScreen();
     this.redirectUrl = '';
+    if (this.filesData && (this.workflowStep === WorkflowStep.UPLOADSTART || this.workflowStep === WorkflowStep.UPLOADING)) {
```

Comment:
If filesData is null/undefined, let's define it here as empty.
```suggestion
    export const WorkflowStep = {
    NOTSTARTED: { key: "not-started", count: -2 }
  UPLOADSTART: { key: "upload-start", count: -3 },
  UPLOADING: { key: "uploading", count: -4 },
};


this.filesData = this.filesData || {};
this.filesData.count = WorkflowStep[this.workflowStep]?.count || -2;
```